domain/relation/types.go

```diff
@@ -342,21 +340,20 @@ type RelationUnitChange struct {
 	// ChangedUnits represents the changed units in this relation.
 	ChangedUnits []UnitChange
 
-	// DepartedUnits represents the units that have departed in this relation.
-	DepartedUnits []int
+	// AvailableUnits represents all the units that are available in the
+	// relation.
+	AvailableUnits []int
 
 	// ApplicationSettings represent the updated application-level settings in
 	// this relation.
 	ApplicationSettings map[string]any
 
-	// Tag is the relation tag that this change relates to.
-	Tag names.Tag
-
-	ApplicationOrOfferToken string
-
-	RelationToken string
-
+	// UnitCount is the total number of units in this relation.
 	UnitCount int
```

Comment:
Do we actually still need this? Without looking, I suspect it goes with ref-counting, which we no longer do.